samples/test/support/management-api/a18nClient.ts

```diff
@@ -38,6 +39,22 @@ class A18nClient {
     return match?.groups?.code;
   }
 
+  async getSmsCode(profileId: string) {
+    let retryAttemptsRemaining = 5;
+    let response;
+    while (!response?.content && retryAttemptsRemaining > 0) {
```

Comment:
fight against sms service?

Reply:
SMS may come with delay, in this case response will have `error` key (with message about empty SMS box) and no `content` key